WalletWasabi.Fluent/ApplicationStateManager.cs

```diff
@@ -191,6 +193,28 @@ private void SetWindowSize(Window window)
 		}
 	}
 
+	private void ObserveWindowSize(Window window, CompositeDisposable disposables)
+	{
+		window?
+			.WhenAnyValue(x => x.Bounds)
+			.Where(b => !b.IsEmpty)
+			.Take(1)
+			.Subscribe(_ =>
+			{
```

Comment:
This is not needed. This was just a trigger for the window having valid bounds.

Reply:
Addition: Trigger for setting the window size.

Reply:
Fixed